### PR TITLE
[Staging] Correcting some spelling and alpha order in new lang strings

### PR DESCRIPTION
**Pull request by infograf768**
As title says
### Summary of Changes
Can be merged on review

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_cpanel.ini [hunk 1]
```diff
@@ -16,6 +16,8 @@ COM_CPANEL_MESSAGES_BODY_NOCLOSE="There are important post-installation messages
...
+COM_CPANEL_MSG_ADDNOSNIFF_BODY="<p>Since Joomla 3.9.3, Joomla is shipped with additional security hardenings in the default htaccess.txt and web.config.txt files. These hardenings disable the so called MIME-type sniffing feature in web browsers. The sniffing leads to specific attack vectors, where scripts in normally harmless file formats (i.e. images) will be executed, leading to Cross-Site-Scripting vulnerabilities.</p><p>The security team recommends to manually apply the necessary changes to existing .htaccess or web.config files, as those files can not be updated automatically.</p><p><strong>Changes for .htaccess</strong><br />Add the following lines before \"## Mod_rewrite in use.\":</p><pre>&lt;IfModule mod_headers.c&gt;\nHeader always set X-Content-Type-Options \"nosniff\"\n&lt;/IfModule&gt;</pre><p><strong>Changes for web.config</strong><br />Add the following lines right after \"&lt;/rewrite&gt;\":</p><pre>&lt;httpProtocol&gt;\n  &lt;customHeaders&gt;\n    &lt;add name=\"X-Content-Type-Options\" value=\"nosniff\" /&gt;\n  &lt;/customHeaders&gt\n&lt;/httpProtocol&gt;</pre>"
+COM_CPANEL_MSG_ADDNOSNIFF_TITLE=".htaccess & web.config security Update"
```
Comment (mbabker) [inline administrator/language/en-GB/en-GB.com_cpanel.ini]: Please make security and Update have consistent casing (as it's a title I'd suggest capitalizing the S).

**Comment by brianteeman** [comment]
> Since Joomla 3.9.3,

Not necessary as the post install message already displays that

> i.e.

The style guide says not to use the punctuation (https://developer.joomla.org/en-gb-user-interface-text-guidelines/punctuation.html)
And personally I think eg is a better fit

Finally if you run this text through something like hemingwayapp.com you will see it has a very high level reading level (lower is better)

File: administrator/language/en-GB/en-GB.com_cpanel.ini [hunk 2]
```diff
@@ -16,6 +16,8 @@ COM_CPANEL_MESSAGES_BODY_NOCLOSE="There are important post-installation messages
...
 COM_CPANEL_MESSAGES_REVIEW="Read Messages"
 COM_CPANEL_MESSAGES_TITLE="You have post-installation messages"
+COM_CPANEL_MSG_ADDNOSNIFF_BODY="<p>Joomla is now shipped with additional security hardenings in the default htaccess.txt and web.config.txt files. These hardenings disable the so called MIME-type sniffing feature in web browsers. The sniffing leads to specific attack vectors, where scripts in normally harmless file formats (eg images) will be executed, leading to Cross-Site-Scripting vulnerabilities.</p><p>The security team recommends to manually apply the necessary changes to existing .htaccess or web.config files, as those files can not be updated automatically.</p><p><strong>Changes for .htaccess</strong><br />Add the following lines before \"## Mod_rewrite in use.\":</p><pre>&lt;IfModule mod_headers.c&gt;\nHeader always set X-Content-Type-Options \"nosniff\"\n&lt;/IfModule&gt;</pre><p><strong>Changes for web.config</strong><br />Add the following lines right after \"&lt;/rewrite&gt;\":</p><pre>&lt;httpProtocol&gt;\n  &lt;customHeaders&gt;\n    &lt;add name=\"X-Content-Type-Options\" value=\"nosniff\" /&gt;\n  &lt;/customHeaders&gt\n&lt;/httpProtocol&gt;</pre>"
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.com_cpanel.ini]:
```suggestion
COM_CPANEL_MSG_ADDNOSNIFF_BODY="<p>Joomla is now shipped with additional security hardenings in the default htaccess.txt and web.config.txt files. These hardenings disable the so called MIME-type sniffing feature in web browsers. The sniffing leads to specific attack vectors, where scripts in normally harmless file formats (eg images) will be executed, leading to Cross-Site-Scripting vulnerabilities.</p><p>The security team recommends to manually apply the necessary changes to existing .htaccess or web.config files, as those files can not be updated automatically.</p><p><strong>Changes for .htaccess</strong><br />Add the following lines before \"## Mod_rewrite in use.\":</p><pre>&lt;IfModule mod_headers.c&gt;\nHeader always set X-Content-Type-Options \"nosniff\"\n&lt;/IfModule&gt;</pre><p><strong>Changes for web.config</strong><br />Add the following lines right after \"&lt;/rewrite&gt;\":</p><pre>&lt;httpProtocol&gt;\n  &lt;customHeaders&gt;\n    &lt;add name=\"X-Content-Type-Options\" value=\"nosniff\" /&gt;\n  &lt;/customHeaders&gt;\n&lt;/httpProtocol&gt;</pre>"
```

Comment (infograf768) [reply administrator/language/en-GB/en-GB.com_cpanel.ini]: Good eyes!

**Comment by HLeithner** [comment]
thx

**Pull request merged**